src/site/content/en/animations/index.njk

```diff
@@ -0,0 +1,8 @@
+---
+layout: path
+override:tags: []
+date: 2020-09-25
+pathName: animations
+tags:
+  - animations
```

Comment:
ah we should remove these. They were causing a bug with our /tags/ landing page.